src/constants.js

```diff
@@ -222,3 +222,8 @@ export const TX_MINING_TESTNET_URL = 'https://txmining.testnet.hathor.network/';
  * Timeout in milliseconds to call the method to set all selected outputs of a tx as 'selected': false
  */
 export const SELECT_OUTPUTS_TIMEOUT = 1000 * 60;
+
+/**
+ * Wallet service URL
+ */
+export const WALLET_SERVICE_BASE_URL  = 'https://dol32y58xi.execute-api.us-east-1.amazonaws.com/production/';
```

Comment:
Maybe load from env variables?

Reply:
We currently have an issue (https://github.com/HathorNetwork/hathor-wallet-lib/issues/161) to improve the way we handle configurations in the wallet. The idea is that we could customize many constants that may depend on the network and we could use env vars to select a custom config file, for example.

I think this comment is valid but it's more complex than just that, so I believe we could keep the current implementation (like we have for the other configurations) for now and focus on that when we implement this issue, what do you think?

Reply:
I just feel that we should use a custom DNS instead of AWS's random DNS.

Reply:
I agree, a custom DNS should be enough

Reply:
Added a TODO in the constants to change this and created an issue in the wallet service repository to create the DNS.